### PR TITLE
Unlock upgrade for KVM release version >= 2.7.0

### PR DESCRIPTION
**Pull request by marians**
See https://gigantic.slack.com/archives/C2MS2VB37/p1537953719000100

This enables upgrades on KVM installations for release version >= 2.7.0.

### CODE REVIEW

File: src/components/organizations/cluster_detail.js [hunk 1]
```diff
@@ -149,7 +144,7 @@ class ClusterDetail extends React.Component {
       return false;
     }
 
-    // cluster release_version must be > 3 on AWS and > 1 on Azure
+    // cluster release_version must be >= 3 on AWS, >= 1 on Azure, >= 2.7.0
     if (this.props.provider === 'aws') {
```
Comment (pipo02mix) [inline src/components/organizations/cluster_detail.js]: I am missing `on KVM`

**Comment by marians** [comment]
I won't merge this before next Monday's product meeting.

**Comment by marians** [comment]
As discussed in the product meeting, upgrades on KVM are still not considered stable.

Unassigning myself here. @giantswarm/team-spirit Can pick this up and merge once the feature is ready for internal use.

**Comment by oponder** [comment]
You need to be an admin anyways before you can see the upgrade button, so we could merge this perhaps?

A long lived PR is kinda like a thorn in my eye ;)

**Comment by marians** [comment]
Please discuss this with team spirit. When we are sure enough that everybody intenrally knows when it's safe to use the button, I'm okay with merging.

**Comment by oponder** [comment]
ping @giantswarm/team-spirit: What's the status of KVM upgrades? Is it ok to expose this feature to us admins in Happa?

**Comment by teemow** [comment]
Yes activating this for admins is fine.

**Comment by calvix** [comment]
+1

**Comment by marians** [comment]
it is merged. I leave the internal communication part to @giantswarm/team-spirit .

**Pull request merged**